lms/djangoapps/grades/tests/test_scores.py

```diff
@@ -106,64 +110,65 @@ def _create_block(self, content_block_value):
         return block
 
     @ddt.data(
-        # submissions _trumps_ other values; weighted and graded from persisted-block _trumps_ latest content values
+        # submissions trumps other values; weighted and graded from persisted-block trumps latest content values
```

Comment:
Nit: Subject-verb agreement - `weighted and graded from persisted-block` **trump** `latest content values`